### DISCUSSION TITLE
performance and accuracy of face recognition compare with original?

### DISCUSSION THREAD

**Discussion by Newbie-Programmer2001** [vladmandic/face-api#172] [Q&A]
is this face-api has improve performance and accuracy of face recognition when compare with original faceapi.js?

**Comment by vladmandic** [comment]
performance? yes. accuracy, no, as learned weights are the same.

**Comment by Newbie-Programmer2001** [comment]
Does this model work for the made Face Attendance System with 20-30 employees?

**Reply by vladmandic** [reply]
that is trivial size.
